test/buildah-bud/apply-podman-deltas

```diff
@@ -193,6 +193,22 @@ skip_if_remote "volumes don't work with podman-remote" \
                "buildah bud --volume" \
                "buildah-bud-policy"
 
+# Most of this should work in podman remote after API implementation other than where context is host.
+skip_if_remote "--build-context option not implemented in podman-remote" \
```

Comment:
Note: I'll open a followup PR to enable `--build-context` on podman-remote since change is big and not relevent to this PR and buildah vendor commit.